projects/control-service/projects/pipelines_control_service/src/main/resources/application.properties

```diff
@@ -99,7 +99,7 @@ logging.level.io.swagger.models.parameters.AbstractSerializableParameter=ERROR
 
 datajobs.control.k8s.namespace=
 datajobs.control.k8s.kubeconfig=${HOME}/.kube/config
-datajobs.control.k8s.k8sSupportsV1CronJob=false
+datajobs.control.k8s.k8sSupportsV1CronJob=true
```

Comment:
Is this explicitly set to false in SuperCollider? Otherwise it might cause some issues.

Reply:
Changed, default in tests only now.